### PR TITLE
Updated samples corresponding to the latest changes

### PR DESCRIPTION
**Pull request by saurabhagas**
Verified all scripts on Mac and Windows.

### CODE REVIEW

**Comment by saurabhagas** [comment]
@jhouserizer I created a `release/10.5` branch so that the older samples can remain intact.

This PR fixes A TON of stuff (several example were broken), while adapting them to dynamic-config.

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,8 @@
 .DS_STORE
-server-data*/
-server-logs*/
```
Comment (mathieucarbou) [inline .gitignore:3]: is it worth keeping the old directory names to reduce the changeset ?

File: 01-ehcache-clustered/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+Important note
+--------------
+Make sure you read and applied the instructions provided [at the top level](../../../)
+
+How to Run
+----------
+
+1. Start a server by running 
+        
+        ./start-server.(bat|sh)
```
Comment (mathieucarbou) [inline 01-ehcache-clustered/README.md:19]: does this take in consideration that now, failover-priority is required ?

File: 01-ehcache-clustered/src/EhCache3ClusteredCache.java [hunk 3]
```diff
@@ -16,12 +33,11 @@
 import java.util.UUID;
 
 public class EhCache3ClusteredCache {
-
   private static final String DEFAULT_TSA_PORT = "9410";
   private static final String TERRACOTTA_URI_ENV = "TERRACOTTA_SERVER_URL";
   private static final String CACHE_MANAGER_ALIAS = "clustered-cache-manager";
   private static final String CACHE_ALIAS = "clustered-cache";
-  private static final String SERVER_RESOURCE = "primary-server-resource";
```
Comment (mathieucarbou) [inline 01-ehcache-clustered/src/EhCache3ClusteredCache.java:24]: is it worth keeping the old directory names to reduce the changeset ?

**Review by mathieucarbou — APPROVED** [review]

**Comment by mathieucarbou** [comment]
I had a look at the changes, they all seem right to me. Approving.
Note: I didn't test all that locally. I guess / hope we have a jenkins job doing that.

File: .gitignore (hunk 1 above)
Comment (saurabhagas) [reply .gitignore:3]: I don't see the worth in doing that. So much has changed, and users better see the new locations for logs, data etc.

**Comment by saurabhagas** [comment]
> I had a look at the changes, they all seem right to me. Approving.
> Note: I didn't test all that locally. I guess / hope we have a jenkins job doing that.

I have tested on Windows and Mac.
There's no CI in this repo (which explains why several samples were broken). The CI job would need to call scripts and check the exit statuses, as there, or we write tests which do that. But this isn't a maven project either. So the work there is non-trivial.

**Pull request merged**